src/app/Fake.Installer.Squirrel/Squirrel.fs

```diff
@@ -46,6 +52,12 @@ type ReleasifyParams = {
 
     /// The secret key for the code signing certificate
     SigningSecret : string option
+
+    /// Set the required .NET framework version, e.g. net461
+    FrameworkVersion : string option
+
+    /// Add other arguments, in case Squirrel adds new arguments in future
+    AdditionalArguments : string list
```

Comment:
It seems like we have any combination of this name already, besides the one you have chosen 💃 

Expecto:
```fsharp
      /// Custom arguments to use in the case the helper not yet supports them
      CustomArgs : string list
```

DotCover:
```fsharp
      CustomParameters: string
```

OpenCover:
```fsharp
          /// This options is used to add additional optional arguments, could be somthing like "-returntargetcode "
          OptionalArguments : string
```

InnoSetup
```fsharp
        /// Additional parameters
        AdditionalParameters : string option
```

Npm/Yarn
```fsharp
    | Custom of string
```

Chocolatey
```fsharp
        /// A character string containing additional arguments to give to choco.
        AdditionalArgs: string
```

I'm not sure but at least we should be consistent and I might enforce a name via contribution guidelines. Do you like any of the existing ones?

Reply:
Also I'm not sure if it should be a list or just a string (and leave escaping to the user).

Reply:
Facepalm for me... I'm sorry.

I think I would prefer AdditionalParameters. The way it is implemented right now, each parameter will be string escaped and surrounded with quotes. Not doing that is easy of course. In that case, I would change it to a string option. Do you have any preference between string option and string list? I can do the change on Sunday or Monday.

Reply:
I was more pointing out the bad quality of the existing code base than the name you have chosen. In fact I think one more name doesn't even matter at this point, but we should work in unifying this in the next version and I'd "lock" the one we choose now in the contribution guidelines :D

There are some tools where the "escaping" will actually break them so I tend to just provide a simple `string` here, but I'm open for arguments for using a list, which I guess would be more "safe".